cmd/signature-v2.go

```diff
@@ -100,6 +100,7 @@ func doesPresignV2SignatureMatch(r *http.Request) APIErrorCode {
 	var gotSignature string
 	var expires string
 	var accessKey string
+	var unescapedQuery string
```

Comment:
any reason to keep this value here, its not used outside of the 'switch' case. ?